index.js

```diff
@@ -1,3 +1,20 @@
-const post = require('./src/post');
+import Post from './src/post';
+import Kpi from './src/kpi';
 
-module.exports = { post };
+export default function DevicePilot(spec) {
+  const { postToken, kpiToken } = spec;
```

Comment:
You're going to need to support `postToken` and `kpiToken` being provided with and without the `TOKEN` prefix. Consider that the app actually copies the word `TOKEN` at the moment, and the documentation isn't clear about removing it (in my experience users will randomly do both).
Easiest way to do this will be to reinstate the configure function (and its tests), and apply it to normalise the tokens provided to your Post and Kpi factories.